pregred-gui/src/app/pages/home/Home.js

```diff
@@ -1,9 +1,255 @@
-import React from 'react';
+import React,{useState} from 'react';
 
 import './Home.css';
+import { Modal, Button } from 'antd';
 
 function Home() {
-  return <h1> PregRed App is Running - Home </h1>;
+  const [visible, setVisible1] = useState(false);
```

Comment:
const should not be general like setVisible1, setVisible2 .. it is a bad practice. naming should be understood

Reply:
sure will edit this, thanks